bintree_test.go

```diff
@@ -13,36 +13,36 @@ func tCheckNode(t *testing.T, tree *binTreeState, index int, state binTreeStateV
 func BenchmarkBinTree(b *testing.B) {
 	tree := binTree{
 		[]binTreeNode{
-			{
```

Comment:
With the changes in the struct definition, these need to change as well. Functionally though, they are the same.